Makefile

```diff
@@ -156,7 +156,7 @@ $(LEGATO_BUILT): $(BUILD_DIR)/.legato_%_built: $(LEGATO_SOURCES_FETCHED) $(TOOLC
 $(LEGATO_SOURCES_FETCHED):
 	rm -rf $(BUILD_DIR)/legato
 	mkdir -p $(BUILD_DIR)/legato
-	cd $(BUILD_DIR)/legato && repo init -u https://github.com/mangOH/manifest.git -m legato/releases/$(LEGATO_VERSION).xml
+	cd $(BUILD_DIR)/legato && repo init -u ssh://gerrit.legato:29418/manifest.git -m legato/releases/$(LEGATO_VERSION).xml
```

Comment:
Is there a reason why you went with gerrit.legato instead of gerrit.legato.io?  Is it because you want to run the script against Legato RC tags?  The downside is that it means that you can't run the script from outside of sierra wireless

Reply:
Yes, it's because of the RC tags.

Reply:
Hopefully we can convince the Legato team to start pushing RC tags to the public Gerrit and/or GitHub, but for now, they don't.